### PR TITLE
fix: redundant flow integration test to unit test

### PR DESCRIPTION
**Pull request by Kelton8Z**

### CODE REVIEW

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: tests/unit/test_clip.py [hunk 1]
```diff
@@ -1,4 +1,90 @@
...
+        assert len(docs.traverse_flat([path]).get_attributes('embedding')) == count
+
+
+def test_custom_processing():
+    f = Flow().add(uses=CLIPImageEncoder)
+    with f:
+        result1 = f.post(on='/test', inputs=[Document(blob=np.ones((224, 224, 3), dtype=np.uint8))],
+                         return_results=True)
+
+    f = Flow().add(uses={
```
Comment (JoanFM) [inline tests/unit/test_clip.py:67]: no Flow should be involved in `unittests`

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — COMMENTED** [review]
Add unittests proving that `docs=None` does not fail

File: tests/unit/test_clip.py (hunk 1 above)
Comment (Kelton8Z) [reply tests/unit/test_clip.py]: fixed

**Review by JoanFM — APPROVED** [review]

**Pull request merged**